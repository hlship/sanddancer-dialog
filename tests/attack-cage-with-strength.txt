(setup lighting)
(grab #strength)
purloin key
teleport break room
attack cage
l
x cage
x door
open cage with key
attack cage
take door
