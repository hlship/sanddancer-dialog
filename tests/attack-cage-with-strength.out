

but it's too late, you're crashing, you crash...

Middle of Nowhere (in the pickup truck)
You've smashed up against a tall Saguaro, which is all you can see out the
windshield except a few feet of desert sand, then blackness. Hell, you must have
driven off the road and crashed. Ocean keeps telling you this night commute on
this lonely highway's gonna kill you; maybe you should start listening.

On the dash you can see a piece of jade.

The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> (setup lighting)
Query succeeded: (setup lighting)
> (grab #strength)
Query succeeded: (grab #strength)
> purloin key

> teleport break room

Break Room
Shadows and grime linger in dark corners of this dismal room with just one tiny
frosted window. Some rotting picnic tables strewn with layers of sand and
crumbling trash are pushed against one wall. On the opposite wall is a locked
wire cage. In the cage is an emergency blanket. A clunky old-fashioned emergency
radio sits on a table.

Open doorways lead west and north.

> attack cage
You grip the mesh in one hand and pull, and realize it's so rusted you can
easily peel it back. The door rips free and you drop it to the floor.

Something about the rusted key makes you want to brood about meeting Ocean.

Hey, that emergency blanket might be useful if you decide on spending the night
here.

> l
Break Room
Shadows and grime linger in dark corners of this dismal room with just one tiny
frosted window. Some rotting picnic tables strewn with layers of sand and
crumbling trash are pushed against one wall. On the opposite wall is a wire cage
that has been ripped open. In the cage is an emergency blanket. A clunky old-
fashioned emergency radio sits on a table.

Open doorways lead west and north.

> x cage
Inside the cage, you see an emergency blanket.

> x door
It's barely recognizable as a door now, just junk.

> open cage with key
There isn't enough of the wire cage left to try that, or any particular reason
to try.

> attack cage
You've vented your frustration on the cage, and won your reward. There isn't
much of it left, anyway.

> take door
You can't take the twisted metal door.

> 
